GREEDY/ImagePreparation/ImageFormating.cs

```diff
@@ -1,171 +1,117 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Drawing;
 using AForge;
 using AForge.Imaging.Filters;
 using Emgu.CV;
-using Emgu.CV.CvEnum;
 using Emgu.CV.Structure;
-using Emgu.CV.Util;
-using Point = System.Drawing.Point;
-using MoreLinq;
-using System.Linq;
+using GREEDY.DataManagers;
 
-namespace GREEDY.DataManagers
+namespace GREEDY.ImagePreparation
 {
     public class ImageFormating : IImageFormating
     {
-        // adds BradleyLocalThreshold filter to the picture
-        public Bitmap BradleyLocalThreshold(Bitmap picture) 
-        {
-            try
-            {
-                Image<Gray, byte> inputImage = new Image<Gray, byte>(picture);
-                BradleyLocalThresholding bradleyLocalThresholding = new BradleyLocalThresholding();
-                Bitmap filtered = new Bitmap(bradleyLocalThresholding.Apply(inputImage.Bitmap));
-                return filtered;
-            }
-            catch
-            {
-                return picture;
-            }
-        }
-
-        private Bitmap Crop(Bitmap bitmap, double[] cornerPoints)
+        /// <summary>
+        /// Applies series of modifications to prepare the image for OCR reading
+        /// </summary>
+        /// <param name="bitmap"></param>
+        /// <returns></returns>
+        public Bitmap FormatImage(Bitmap bitmap)
         {
-            // define quadrilateral's corners
-            List<IntPoint> corners = new List<IntPoint>
-            {
-                new IntPoint(Convert.ToInt32(cornerPoints[0]), Convert.ToInt32(cornerPoints[1])),
-                new IntPoint(Convert.ToInt32(cornerPoints[2]), Convert.ToInt32(cornerPoints[3])),
-                new IntPoint(Convert.ToInt32(cornerPoints[4]), Convert.ToInt32(cornerPoints[5])),
-                new IntPoint(Convert.ToInt32(cornerPoints[6]), Convert.ToInt32(cornerPoints[7]))
-            };
-            QuadrilateralTransformation filter = new QuadrilateralTransformation(corners);
-            Bitmap newImage = filter.Apply(bitmap);
-            return newImage;
+            Bitmap edited;
+            edited = Binarization(bitmap);
+            edited = RemoveNoise(edited);
+            edited = BiggestBlob(edited);
+            edited = Rotate(edited);
+            edited = Deskew(edited);
+            edited = BiggestBlob(edited);
+            edited = Rescale(edited);
+            return edited;
         }
 
-        //returns the bigest found rectangle
-        private Bitmap FilterCropedImages(List<Bitmap> list)
+        //Not really sure what it does but the recomendations suggest it helps the OCR read better
```

Comment:
"Tesseract works best on images which have a DPI of at least 300 dpi, so it may be beneficial to resize images" official Tesseract page. so, you have to change comment, because we know that it does and why we need it :)